Jenkinsfile

```diff
@@ -99,5 +108,44 @@ pipeline {
         """
       junit testResults: "logs/junit.xml", keepLongStdio: false
     }
+    success {
+      script {
+        if( env.OCS_OPERATOR_DEPLOYMENT == true || env.OCS_OPERATOR_DEPLOYMENT == "true" ) {
+          def messageContent = '''
+          {
+            "SENDER_BUILD_NUMBER": "${BUILD_NUMBER}",
+            "OCS_OPERATOR_IMAGE": "${env.OCS_OPERATOR_IMAGE}"
+          }
+          '''
+        } else {
+          def messageContent = '''
+          {
+            "SENDER_BUILD_NUMBER": "${BUILD_NUMBER}",
+            "ROOK_IMAGE": "${ROOK_IMAGE}",
+            "CEPH_IMAGE": "${CEPH_IMAGE}",
+            "CEPH_CSI_IMAGE": "${CEPH_CSI_IMAGE}",
+            "ROOK_CSI_REGISTRAR_IMAGE": "${ROOK_CSI_REGISTRAR_IMAGE}",
+            "ROOK_CSI_PROVISIONER_IMAGE": "${ROOK_CSI_PROVISIONER_IMAGE}",
+            "ROOK_CSI_SNAPSHOTTER_IMAGE": "${ROOK_CSI_SNAPSHOTTER_IMAGE}",
+            "ROOK_CSI_ATTACHER_IMAGE": "${ROOK_CSI_ATTACHER_IMAGE}",
+          }
+          '''
+        }
+        if( env.UMB_MESSAGE == true || env.UMB_MESSAGE == "true" ){
+          sendCIMessage \
```

Comment:
can you use syntax with brackets rather than backslashes? sendCIMessage( //break the line here without need of \